ae/templates/engine-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+{{- with .Values.engine.secrets }}
```

Comment:
Can you please follow the same "convention" as the one we had for the cockpit certificates ?

https://github.com/gravitee-io/helm-charts/blob/master/apim/3.x/templates/cockpit/cockpit-certificates.yaml#L1

Reply:
So basically, having the license not at the engine level, but at the root level.
Also, this is interesting for AE, but that would be awesome to replicate this for APIM / AM

Reply:
Can we also take into consideration that we can use the same secret for AE, AM and APIM as long as they are in the same namespace?
So we could check if the secret already exists before we create it.

Reply:
Yes good remarks, and thanks for your feedback, I am working on it.

Reply:
✔ I let you keep a fresh eyes on it.
Thank you !